### ISSUE TITLE
Pandoc 2.0.4 seemingly recognised as a 1.x version

### ISSUE THREAD

**Issue by arthur-st** [kawabata/ox-pandoc#47]
Running Spacemacs 0.200.10 in Emacs 25.3 ( https://github.com/zklhp/emacs-w64/releases ) on Windows 10 (x64). Pandoc is enabled in `dotspacemacs-configuration-layers`, and I've installed Pandoc 2.0.4.

Despite `(setq pandoc-binary "C:\\Tools\\Pandoc\\pandoc.exe")` in my user config, I get `Warning (ox-pandoc): This Pandoc (1.x) may not support new pandoc features.` message on startup.

**Comment by vv111y** [comment]
I'm getting this error as well
fwiw
#### System Info :computer:
- OS: darwin
- Emacs: 26.0.91
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 0fa3658cd)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm

**Comment by vv111y** [comment]
I'm on a mac so this https://github.com/syl20bnr/spacemacs/issues/2927 seems more appropriate for me.  
It's a conflict with Anaconda installing it's own version of Pandoc

**Comment by aurghyadip** [comment]
I agree with @vv111y , I am on a windows machine and it was a conflict with my Anaconda Installation. After I removed Anaconda from the path, everything was normal.
However, in my Ubuntu 18.04 machine (anaconda installed) I did not get the problem.

**Comment by QikeLi** [comment]
The following command solved my problem:
conda install -c conda-forge pandoc
